Makefile

```diff
@@ -33,48 +34,35 @@ clean: ## cleans up the repository
 	/bin/rm -rf $(PKGDIR)
 	/bin/rm -rf $(DATADIR)
 
-build: $(addprefix build/,$(DISTRIBUTIONS)) ## builds the code for all distributions
-
-test: $(addprefix test/,$(DISTRIBUTIONS)) ## runs tests for all distributions
-
-test/%: TEST_FLAGS ?= -v
-test/%: vet ## runs unit tests for a specific distribution
-	@echo "*\n* Testing for $* locally \n*"
-	go test $(TEST_FLAGS) -tags $*
+test: vet 
+	go test -v
+	$(MAKE) -C osDiscovery test
+	$(MAKE) -C packageLookup test
 
 format: ## formats the code
 	gofmt -w $(SOURCE)
 
 vet: ## examines the go code with `go vet`
-	go vet $(SOURCE)
+	go vet
 
 up: $(addprefix up/,$(DISTRIBUTIONS)) ## start agents for all distributions
 up/%: ## starts the agent for a specific distribution
 	docker-compose --project-name=tactycal up agent$*
 
 run/%:
```

Comment:
fixed. `/%` has been removed. `docker-compose.ym` and `docker-compse.tmpl` files have been also updated.